tests/python/gaia-ui-tests/gaiatest/gaia_test.py

```diff
@@ -142,6 +142,22 @@ def kill_all(self):
         # Now kill the user apps
         self.marionette.switch_to_frame()
         self.marionette.execute_async_script("GaiaApps.killAll();")
+    
+    def install(self, manifest_url):
```

Comment:
install an uninstall share a lot in common. How about reducing the duplication, this way:
```py
def install(manifest_url):
    self._change_state_of_app(manifest_url, 'install')

def uninstall(manifest_url):
    self._change_state_of_app(manifest_url, 'uninstall')

def _change_state_of_app(manifest_url, desired_action):
    self.marionette.switch_to_frame()
    data_layer = GaiaData(self.marionette)
    data_layer.set_bool_pref('dom.mozApps.auto_confirm_{}'.format(desired_action), True)
    result = self.marionette.execute_async_script('GaiaApps.{}WithName("{}")'.format(manifest_url, desired_action))
    assert (result is True), 'Failed to {} app: {}'.format(manifest_url, desired_action)
    data_layer.set_bool_pref('dom.mozApps.auto_confirm_{}'.format(desired_action), False)
```